recipe/meta.yaml

```
@@ -24,78 +24,76 @@ requirements:
     - python-build
     - meson-python ==0.13.1
     - meson ==1.2.1
-    - cython >=0.29.33,<3
+    - cython >=3.0.5
     - numpy {{ numpy }}
```

```suggestion
    - numpy 1.22 # [py<311]
    - numpy {{ numpy }} # [py>=311]
```
Because of https://github.com/pandas-dev/pandas/blob/v2.2.1/pyproject.toml#L33

---

Done!